<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    <meta name="viewport" content="width=device-width, user-scalable=no">

    <title>Research Interests//Jean-David Grattepanche</title>
    <meta name="description" content="Jean-David Grattepanche aims to assess the diversity and ecology of microbial eukaryotes in marine systems using microscopy, molecular tools and bioinformatics" />
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
    <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
    <link rel="manifest" href="/manifest.json">
    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
    <meta name="theme-color" content="#ffffff">
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
    <link href="/stylesheets/normalize.css" rel="stylesheet" /><link href="/stylesheets/all.css" rel="stylesheet" /><link href="/stylesheets/animate.min.css" rel="stylesheet" />
    <script src="https://use.fontawesome.com/c35cf0bf29.js"></script>
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-83201064-1', 'auto');
      ga('send', 'pageview');

    </script>
  </head>

  <body class="research">
    <nav id="main-menu" class="navbar navbar-default navbar-fixed-top shadow">
   <div class="container">
      <div class="navbar-header">
         <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
         <span class="sr-only">Toggle navigation</span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
         </button>
         <a class="navbar-brand" href="/">J-D G.</a>
      </div>
      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
         <ul class="nav navbar-nav navbar-right">
            <li><a href="/">
              <i class="fa fa-home" aria-hidden="true"></i> Home
</a>            </li>
            <li><a href="/resume.html">About me</a></li>
            <li><a href="/publications.html">Publications & Communications</a></li>
            <li><a href="/research.html">Research Interests</a></li>
            <li><a href="/phd-abstract.html">PhD Abstract</a></li>
            <li><a target="_blank" href="http://jeandavidgrattepanche.weebly.com/contact-me.html">Contact</a></li>
         </ul>
      </div>
   </div>
</nav>



    <section>
  <div class="mini-header">
    <div class="header-title wow fadeIn">
      <h1 class="shadow">Research Interests</h1>
    </div>
  </div>
</section>

<section>
  <div class="container ptb wow fadeIn">
    <h2>Statement of Research Interests and Career Goals</h2>
    <p class="big-text text-left gradient-text ptb-30">&emsp;Microbes constitute the bulk of the biodiversity and biomass on Earth and have critical impacts on the function of ecosystems. In order to understand relationships between microbial eukaryotes within the planktonic food web, my goals are to characterize the diversity and spatiotemporal variability of the microbial community. This includes, but is not limited to, examining the trophic links between microzooplankton and both phyto- and zooplankton. I also aim to capture species-specific relationships within the planktonic food web through analyses of prey-predator interactions and microcosm experiments. I use a combination of microscopy, molecular tools (cloning, a DNA fingerprinting technique [Denaturant Gradient Gel Electrophoresis: DGGE], and high throughput sequencing), and bioinformatics to explore these questions. Together, these approaches increase our understanding of the dynamics underlying the microbial portion of marine food webs.</p>
    <p class="ptb-30 gradient-text">
      <b>Keywords</b>: Ecology, Biodiversity, Planktonic food web, microbial eukaryotes, SAR clade
    </p>
  </div>
  <div class="container">
    <div class="col-xs-12 col-sm-8 col-sm-offset-2
     ptb wow fadeInUp">
      <h2>1.Planktonic food web</h2>
      <p>&emsp;Since Pomeroy (1974) suggested the role of micro-heterotrophs, it became increasingly clear that protozoans play an important ecological role in the planktonic food web. This role is linked to their central position in the trophic food web as protozoans are intermediate between classic food chain and the microbial food web (Sherr & Sherr, 1986; Gifford, 1991). Microplanktonic protozoans are predators of bacteria (Fenchel, 1982; Sieburth & Davis, 1982; Azam et al., 1983), phytoplankton (Calbet & Landry, 2004), other microzooplanktonic organisms, such as protozoans and copepod nauplii, (Merrel and Stoecker, 1998; Jeong, 1999) and even aggregates (Silver et al.1984, Grossart and Simon, 1993). These micro-heterotrophs also play a role as a food source for zooplankton (copepods, rotifers, reviewed in Stoecker and Capuzzo, 1990; Gifford,1991), improving their production and survival rate (Tang et al.2001; Klein Breteler et al., 2003). Therefore, depending on the studied area, copepods can consume from 0 to 200% of the ciliates stock per day (Dolan, 1991; Nielsen & Kiørboe, 1994; Lonsdale et al.,2000; Koski et al., 2002).</p>
    </div>
    <div class="col-xs-12 col-sm-8 col-sm-offset-2 wow fadeInUp ptb-30" >
      <h3>1-1.Temporal variability of protist communities</h3>
      <p>&emsp;During my master’s degree and my PhD, the community structures and succession of phytoplankton, protozooplankton and mesozooplankton (copepods) s were studied from February 2007 to July 2009 in a coastal area of the eastern English Channel in which recurrent Phaeocystis globosa blooms occur. This survey allowed to identify main microplankton assemblages as well as to define interesting periods in terms of the microbial web functioning. My researches highlighted : (1) the seasonal variability of heterotrophic protists in strong relation to  the phytoplankton succession (bottom-up control); (2) the year to year variability of micrograzers in relation to P. globosa bloom magnitude and duration; and (3) the importance of dinoflagellates as major consumers of phytoplankton, particularly of diatoms and P. globosa colonies ( < 100 μm). This in situ survey also suggested the strong top-down control exerted by copepods on heterotrophic protists. The “grazing hypotheses” drawn during this first part of my PhD were then studied in two different types of experiments.
      </p>
    </div>
    <div class="row">
      <div class="col-xs-12 col-md-6 wow fadeInUp ptb-30">
        <h3>1-2.Estimation of Protozooplankton herbivory via dilution experiments</h3>
        <p>&emsp;The dilution technique was used to investigate microzooplankton grazing and phytoplankton growth in the eastern English Channel during the diatom–Phaeocystis spring succession from January to June 2009. Dilution experiments and size-fractionated dilution experiments were conducted before, during and after the P. globosa bloom and confirmed in situ hypotheses. The size classes (i.e. <5 µm; 5-10 µm and >10 µm) were chosen as they match the size spectra exhibited by the distinct life stages of P. globosa i.e. 5-10µm Chla corresponded to colonial and large flagellated cells of P. globosa whereas the <5 µm Chla essentially corresponded to small flagellated cells. Microzooplankton carbon consumption (from 18.1 to 360.9 µg C L-1 d-1) often equaled or exceeded phytoplankton production (from 1.7 to 129.0 µg C L-1 d-1) in particular at the end of the P. globosa bloom when microzooplankton grazed on previously formed phytoplankton biomass. Results of size-fractionated dilution experiments, conducted with distinct grazer communities, suggested different roles for ciliates and dinoflagellates. Ciliates appeared to be very efficient grazers of small diatoms (5-10 µm) and P. globosa free cells, whereas dinoflagellates grazed on both larger diatoms (> 10 µm P1) and small P. globosa colonies. Ciliates and dinoflagellates did not seem to compete for food resources, as they were oriented towards different phytoplankton size classes.</p>
      </div>
      <div class="col-xs-12 col-md-6 wow fadeInUp text-center ptb">
        <img src="images/phd.jpg" alt="Model of the fluxes estimated and observed during the In situ survey and microcosm experiment in the Eastern English Channel" class="ptb-30 img-responsive plr">
        <p><small>Model of the fluxes estimated and observed during the In situ survey and microcosm experiment in the Eastern English Channel (Grattepanche 2011, PhD defense)</small></p>
      </div>
    </div>

    <div class="col-xs-12 col-sm-8 col-sm-offset-2 wow fadeInUp ptb-30">
      <h3>1-3.Zooplankton grazing impact on protozooplankton via a microcosm experiments </h3>

      <p>&emsp;The importance of heterotrophy in the eastern English Channel was estimated in dark microcosms experiment during 19 days. By excluding physical forcing and nutrient input, this microcosm study confirmed the role of protozooplankton as major consumers of phytoplankton during the Phaeocystis globosa bloom, especially before the establishment of zooplankton metazoans. Three periods were defined during the microcosm experiment based on phytoplankton and microzooplankton dynamics: Period Pphyto corresponding to phytoplankton growth; Period Pmicroz characterized by the dominance of protozooplankton and Period Pmetaz dominated by metazoans such as rotifers and copepods. During Pphyto and Pmicroz, the major part of phytoplankton was consumed by protozooplankton. Micro-heterotrophs grazing pressures (% biomass consumed per day) reached 10-25% d-1 for nanophytoplankton and 15-20% d-1 for < 100µm diatoms. Rotifers and copepods were identified as major phytoplankton and protozooplankton consumers during Pmetaz. Comparison with the in situ survey showed that metazoan’s predation could be the main control of protozooplankton during the wane of the P. globosa bloom. Metazoans peaked one week delay after protists, suggesting time lag response to available trophic resource for these two groups. This experiment allowed estimating the metazoan impact (particularly rotifers) on microplanktonic protozoans (predation/competition).</p>
    </div>

    <div class="col-xs-12 col-sm-8 col-sm-offset-2 wow fadeInUp ptb">
      <h2>2.Diversity of plankton</h2>
      <p>&emsp;A deeper understanding of species-specific links within the planktonic food webs is related to both the spatial scale and ecological drivers of microbial diversity. As a postdoctoral fellow at Smith College, I have been studying the diversity of ciliates, a clade of microbial eukaryotes, to assess how biodiversity varies with abiotic and biotic factors. Using the DNA fingerprinting technique denaturant gradient gel electrophoresis (DGGE), we observed that changes in ciliate community structure at small scales relate to the tides (Grattepanche et al 2014a), the presence of a large scale community ranging almost everywhere between the coast of Rhode Island and the continental shelf break (160km), and throughout the water column, from the surface to our deepest samples (up to 850 m; Grattepanche et al 2015).</p>
    </div>
    <div class="ptb-30 text-center col-xs-12 col-sm-8 col-sm-offset-2 wow fadeInUp">
      <img src="images/re_interest3.png" alt="Ciliate assemblages observed off the coast of the New England using DGGE" class="img-responsive">
      <small>Ciliate assemblages observed off the coast of the New England using DGGE (Grattepanche et al 2015)</small>
    </div>

    <div class="col-xs-12 col-sm-8 col-sm-offset-2 ptb wow fadeInUp">
      <p>&emsp;While patterns of abundant community members can be observed by using “classic” tools such as microscopy, clone libraries or DGGE in our case (Grattepanche et al 2014a, 2015), high throughput sequencing (HTS) technologies allow observations of patterns at finer taxonomic scales. Among the more dramatic insights from HTS has been the discovery of numerous rare lineages of both bacteria and archaea in natural environments (i.e. the rare biosphere; Sogin et al 2006) as well as tremendous diversity of eukaryotes (e.g. Santoferrara et al 2014, 2016; Grattepanche et al 2014b, 2016, submitted).  Assessing the diversity of eukaryotic microbes requires accounting for both the technical limits (e.g. bias, accuracy, sample size) and the heterogeneity in units of biodiversity (e.g. species) that vary based on analyses of molecules, morphology and behavior (Grattepanche et al 2014b).</p>
    </div>

    <div class="col-xs-12 col-sm-6 col-sm-offset-3 text-center ptb wow fadeInUp">
      <img src="images/re_interest1.png" alt="Distribution of OTU richness" class="img-responsive">
      <small>Distribution of OTU richness for each sample shows a complex pattern but overall a relatively high number of OTUs within the deep samples compared to the sample taken above (Grattepanche et al 2016).</small>
    </div>

    <div class="row">
      <div class="col-xs-12 col-sm-6 wow fadeInUp ptb">
        <p>&emsp;Analyses of HTS data on ciliates from coastal waters in New England reveals an increase of diversity with the depth, particularly below the photic zone (Grattepanche et al 2015, 2016, submitted), which contradicts previous estimates based on microscopy and clone libraries (e.g. Christaki et al 2011, Wickham et al 2011). RNA/DNA comparisons using DGGE show that this diverse communities observed below the photic zone are active (Tucker et al in prep, Grattepanche et al in prep). These HTS analyses also provide evidence for two novel Spirotrichea lineages, named Clusters X and Cluster Y, which lack morphological information (Grattepanche et al 2016). I am currently using FISH (Fluorescence In Situ Hybridization) to identify these ciliates from marine samples. In my previous studies, we observed that the pattern of communities is generally not well connected to the environmental parameter. Analyses of ciliate communities at small scales (1-3 km; Grattepanche et al submitted) shows that individual OTU show specific biogeographies particularly for abundant OTUs related to the depth (surface, chlorophyll maximum, deep) and to the position on the shore.</p>
        <p>&emsp;​While I continue to explore the spatial scale of ciliate diversity, I have begun to use an experimental approach to elucidate the processes that drive these patterns.  For example, I created microcosms, small scale replicates of marine environments that can be subject to varying environmental changes, to assess how bottom-up and top-down controls impact the ciliate community. Our first set of microcosm experiments showed that copepod decrease ciliate diversity. Another experiment, which is in progress, investigates the effect of phytoplankton change on ciliate communities (bottom-up control), and aims to mimic the impact of algal blooms on ciliates communities (Juarez et Al in prep).</p>
        </div>
      <div class="col-xs-12 col-sm-6 text-center wow fadeInUp ptb">
        <img src="images/re_interest2.png" alt="Heterogeneous patterns of OTUs" class="img-responsive">
        <small>Heterogeneous patterns of OTUs (i.e. OTU specific to the inshore or<br> to the offshore area, to the photic zone or to the deep) <br> mapped onto phylogenetic tree (Grattepanche et al, submitted).</small>
      </div>
    </div>

    <div class="col-xs-12 col-sm-8 col-sm-offset-2 wow fadeInUp ptb">
      <h2>3.Future directions</h2>
      <div class="text-center ptb">
        <h3>- Diversity patterns -</h3>
      </div>
      <h4>Microbial eukaryotes below the photic zone</h4>
      <p>&emsp;Contrary to conventional wisdom, my diversity studies using DGGE and HTS reveal an incredible diversity of small ciliates in the deep ocean environment (up to 850m of depth; Grattepanche et al 2015, submitted). I will conduct additional analyses of this extreme environment, which I predict will lead to the discovery of a large number of previously-unknown microbial eukaryotic lineages. I plan to increase my taxonomic focus to understudied microbial lineages such as Foraminifera and Radiolaria. I will sample microbial communities at varying depths below the photic zone to elucidate members and patterns in the deep-water environment.  I will also analyze patterns in both DNA and RNA to understand what proportion of the community is most active. Knowledge of these organisms will allow us to have a better picture of the biodiversity on Earth and may fill some gaps in the eukaryotic tree of life. It is even possible that some of these deep-water microbes could represent ‘living fossils’ and thus will help us to understand more about the Last Eukaryote Common Ancestor (LECA).</p>
      <div class="text-center ptb">
        <h3>- Linking diversity to ecology -</h3>
      </div>
      <div class="ptb-30">
        <h4>Interdisciplinary approaches to understanding impact of climate change on microbial diversity</h4>
        <p>&emsp;I will combine my expertise in HTS and experimental (e.g. microcosm) studies to explore how the diversity of the microbial eukaryotic community varies in response to conditions that mimic climate change. The global warming will affect abiotic parameters such as temperature, acidity, and speed of thermohaline circulation, and biotic factors such as magnitude of harmful algal blooms. Of particular interest is to understand how and at which magnitude the microbial diversity will be affected by these changes. These studies will require an interdisciplinary approach to: (1) identify species-specific impacts on biogeochemical cycles, (2) capture the abiotic variation within microhabitats and (3) understand what affects microbial diversity at varying spatial and temporal scales.</p>
      </div>
      <div class="ptb-30">
        <h4>Metatranscriptomic approach</h4>
        <p>&emsp;Another direction that I intend to pursue is the study of the ecological function of plankton communities. Metatranscriptome allows for investigation of the active genetic content of the planktonic food web in various ecosystems.  Such analyses will reveal the ecological functions present in the ecosystem and aid in identifying new genes of ecological and biotechnological importance (e.g. toxin production); SIP-FISH, which combines FISH and Stable Isotope Probing, and in vitro experimentation, can be powerful tools for elucidating ecological function. While SIP-FISH is yet to be developed for microbial eukaryotes, this kind of approach will allow the tracking of gene expression, which can be related back to ecosystem function. Overall, metatranscriptomics will serve as one essential component for assessing ecosystem health.</p>
      </div>
    </div>


    <div class="col-xs-12 col-sm-8 col-sm-offset-2 ptb-30 wow fadeInUp">
      <h2>4.Career Goals</h2>

      <p>&emsp;My career goals are to expand my knowledge on heterotrophic protist diversity and its impact on the trophic food web in various areas. In addition, as a young researcher I wish to broaden my research perspectives and opportunities without limiting myself to geographical considerations.</p>

      <div class="text-center ptb-60">
        <img src="images/target.svg" alt="career goals">
        <div class="wow bounceInUp ptb-30">
          <p class="gradient-text">You need more informations?</p>
          <br>
          <a href="http://jeandavidgrattepanche.weebly.com/contact-me.html" class="link-btn" target="_blank">Get in Touch</a>
        </div>
      </div>
    </div>


  </div>
</section>




    <footer id="footer">
   <div class="container ptb-30">
      <hr/>
      <div class="space-15"></div>
      <div class="row">
        <div class="col-xs-12 col-sm-6">
          <p>&copy; 2017 Jean-David Grattepanche. All Rights Reserved.</p>
        </div>
        <div class="col-xs-12 col-sm-6 text-right">
          <ul class="list-unstyled list-inline">
            <li><a href="mailto:jeandavid.grattepanche@gmail.com"><i class="fa fa-envelope" aria-hidden="true"></i></a></li>
            <li><a href="https://twitter.com/JDGrattepanche?lang=fr"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
          </ul>
        </div>
      </div>
      <a href="http://guillaumecaulier.com" target="_blank" id="gui"><i class="fa fa-code" aria-hidden="true"></i> by Guillaume Caulier</a>
   </div>
</footer>

    <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
    <script src="/javascripts/all.js"></script>
  </body>
</html>
